src/env/parsers.test.ts

```diff
@@ -53,3 +53,24 @@ describe('oneOf', () => {
       `"process.env.ENVIRONMENT is not a supported choice: 'dev'. Expected one of: ['local', 'prod']"`,
     ));
 });
+
+describe('boolean', () => {
+  const happyCases = [['1'], ['true'], ['yes'], ['on'], ['enabled']] as const;
+
+  it.each(happyCases)('parses %s', (input) =>
```

Comment:
I think Jest has a cheat here

```suggestion
  const happyCases = ['1', 'true', 'yes', 'on', 'enabled'];

  it.each(happyCases)('parses %p', (input) =>
```